packages/loot-core/src/client/privacy.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { createContext, useContext } from 'react';
+import { useSelector } from 'react-redux';
+
+type TPrivacyModeContext = {
+  enabled: boolean;
+};
+
+let PrivacyContext = createContext<TPrivacyModeContext>(null);
+
+export function PrivacyProvider({ children }) {
+  let prefs = useSelector(state => state.prefs);
+  let isPrivacyEnabled = prefs?.local?.isPrivacyEnabled || false;
+
+  return (
+    <PrivacyContext.Provider
+      value={{ enabled: isPrivacyEnabled }}
+      children={children}
+    />
+  );
+}
+
+export function usePrivacyMode() {
+  let context = useContext(PrivacyContext);
+  return context?.enabled || false;
```

Comment:
Could you update this to directly call `useSelector` or did you see bad performance doing something like this?

```js
return useSelector(state => state.prefs?.local?.isPrivacyEnabled ?? false)
```

Reply:
No performance problems, I just patterned it to how the other context hooks are done throughout the codebase.

Reply:
Does this need to be in context? It looks like the other context providers generally do something a bit more complicated than just read a value from Redux, so I think here it would make sense to avoid adding the wrapping context object and instead use the existing Redux structure.

Reply:
Removed the `PrivacyProvider` in favor of directly accessing the privacyMode flag via `useSelector`.